lv_conf_template.h

```diff
@@ -113,6 +113,11 @@
 /*The target buffer size for simple layer chunks.*/
 #define LV_DRAW_LAYER_SIMPLE_BUF_SIZE    (24 * 1024)   /*[bytes]*/
 
+/* The stack size of the drawing thread.
+ * NOTE: If FreeType or ThorVG is enabled, it is recommended to set it to 32KB or more.
+ */
+#define LV_DRAW_THREAD_STACK_SIZE    (8 * 1024)   /*[bytes]*/
```

Comment:
It's related only to the SW render.  It'd be better to rename it to `LV_DRAW_SW_THREAD_STACKSIZE` and add a compatibility define in `lv_conf_internal.h`:
```c
#ifdef LV_DRAW_THREAD_STACK_SIZE
#warning "LV_DRAW_THREAD_STACK_SIZE was renamed to LV_DRAW_SW_THREAD_STACK_SIZE. Please update lv_conf.h or run menuconfig again."
#define LV_DRAW_SW_THREAD_STACK_SIZE LV_DRAW_THREAD_STACK_SIZE
#endif
```